account_financial_report/static/src/js/account_financial_report_backend.js

```diff
@@ -4,17 +4,22 @@ odoo.define("account_financial_report.account_financial_report_backend", functio
     "use strict";
 
     var core = require("web.core");
-    var Widget = require("web.Widget");
-    var ControlPanelMixin = require("web.ControlPanelMixin");
+    var AbstractAction = require('web.AbstractAction');
     var ReportWidget = require("account_financial_report.account_financial_report_widget");
 
-    var report_backend = Widget.extend(ControlPanelMixin, {
+    var report_backend = AbstractAction.extend({
+        // TODO: why isnt the controlpanel shown?
+        hasControlPanel: true,
```

Comment:
@NL66278 Despite all of my attempts here, the control panel is still not shown. Do you know what is going wrong?

Reply:
@thomaspaulb I have no idea I do have only an extremely limited experience with javascript in Odoo...